### PR TITLE
Add `paywallViewControllerDidStartPurchase` to wrapper delegate

### PR DESCRIPTION
**Pull request by vegaro**
Added new `paywallViewControllerDidStartPurchase`

### CODE REVIEW

File: ios/PurchasesHybridCommon/PurchasesHybridCommonUI/PaywallProxy.swift [hunk 1]
```diff
@@ -183,6 +183,10 @@ import UIKit
 
 @available(iOS 15.0, *)
 extension PaywallProxy: PaywallViewControllerDelegate {
+    
+    public func paywallViewControllerDidStartPurchase(_ controller: PaywallViewController) {
```
Comment (tonidero) [inline ios/PurchasesHybridCommon/PurchasesHybridCommonUI/PaywallProxy.swift:187]: Do we also need one for restore starting?

**Review by tonidero — APPROVED** [review]
This looks good! Just a question

File: ios/PurchasesHybridCommon/PurchasesHybridCommonUI/PaywallProxy.swift (hunk 1 above)
Comment (vegaro) [reply ios/PurchasesHybridCommon/PurchasesHybridCommonUI/PaywallProxy.swift:187]: yes, but that one is not available in purchases-ios yet so we need to add it there first 😢

File: ios/PurchasesHybridCommon/PurchasesHybridCommonUI/PaywallProxy.swift [hunk 2]
```diff
@@ -183,6 +183,10 @@ import UIKit
...
+        self.delegate?.paywallViewControllerDidStartPurchase?(controller)
```
Comment (NachoSoto) [inline ios/PurchasesHybridCommon/PurchasesHybridCommonUI/PaywallProxy.swift:188]: Thanks for adding this!

**Pull request merged**